.github/workflows/format-lint.yml

```
@@ -0,0 +1,31 @@
+  pull_request:
+    branches: [ '**' ]
+  workflow_dispatch:
+
+jobs:
+  format-and-lint:
+    name: Format and Lint
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)